pkg/system/phase4_configuring.go

```diff
@@ -94,6 +94,9 @@ func (r *Reconciler) ReconcilePhaseConfiguring() error {
 	if err := r.ReconcileReadSystem(); err != nil {
 		return err
 	}
+	if err := r.ReconcileDefaultNamespaceStore(); err != nil {
```

Comment:
move this before the ReconcileDefaultBucketClass. also, you are missing on the code that should put the default namespace store inside of the default bucket class.

Reply:
Yes, that is the plan. Will change it.